Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -99,9 +102,9 @@
 from FWCore.Modules.logErrorHarvester_cfi import *
 
 # "Export" Section
-reconstruction         = cms.Sequence(localreco*globalreco*highlevelreco*logErrorHarvester)
+reconstruction         = cms.Sequence(localreco*globalreco*highlevelreco*cosmicCDCTracksSeq*logErrorHarvester)
```

Comment:
please remove cosmicCDCTracksSeq from reconstruction\* sequence definitions.
It should be moved to highlevelreco (I don't think it fits well in the globalreco).